app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,66 @@
  *= require_tree .
  *= require_self
  */
+
+.splash {
+  display: flex;
+  flex-direction: column;
+}
+
+.splash h1 {
+  margin-bottom: 80%;
+  color: red;
+  margin-top: 10%;
+}
+
```

Comment:
- [ ] According to the original design the name of the app is black in color when the whole background color is white. Please try and fix that and the font family as well.
- [ ] There is a large gap between the name of the app and the signup and login buttons. Please try and fix that

Reply:
Thanks for the review, I have made the necessary changes. There is a considerable gap in the original design, however I have reduced mine

Reply:
This changes have been done well